### PR TITLE
Fix math for bearer token refresh time

### PR DESCRIPTION
**Pull request by sjoubert**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [ ] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [ ] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

**Comment by antkmsft** [comment]
Thank you @sjoubert!

**Review by antkmsft — APPROVED** [review]

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-beta.6 (Unreleased)
 
+### Bug Fixes
+
+- Fixed computation of the token expiration time in `BearerTokenAuthenticationPolicy`
```
Comment (antkmsft) [inline sdk/core/azure-core/CHANGELOG.md:7]:
```suggestion
- Fixed computation of the token expiration time in `BearerTokenAuthenticationPolicy`.
```

**Comment by vhvb1989** [comment]
@antkmsft can we create an issue to add tests for this at some point?

**Comment by ahsonkhan** [comment]
@vhvb1989 see reference link above. It's already there: https://github.com/Azure/azure-sdk-for-cpp/issues/1596

**Pull request merged**